common/src/commonMain/kotlin/io/xorum/codeforceswatcher/network/responses/ContestsResponse.kt

```diff
@@ -1,10 +1,41 @@
 package io.xorum.codeforceswatcher.network.responses
 
+import com.soywiz.klock.DateFormat
+import com.soywiz.klock.parse
 import io.xorum.codeforceswatcher.features.contests.models.Contest
+import io.xorum.codeforceswatcher.features.contests.models.Platform
+import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class ContestsResponse(
+data class ContestResponse(
+        val name: String,
+        val url: String,
+        @SerialName("start_time") val startTime: String,
+        val duration: Double,
         val status: String,
-        val result: List<Contest>
-)
\ No newline at end of file
+        @SerialName("site") val platform: String
+) {
+    fun toContest(): Contest {
+        val dateFormat = DateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
+        println("Here : here : $startTime + ${dateFormat.parse(startTime).local.unixMillisLong} + ${platform}")
+        return Contest(0L, name, dateFormat.parse(startTime).local.unixMillisLong, duration.toLong() * 1000, status, parsePlatform(platform)).apply { link = url }
+    }
+
+    private fun parsePlatform(platform: String): Platform {
```

Comment:
Can be written as `private fun parsePlatform(platform: String) = when(platform) {`